JsonSubTypes/JsonSubTypes.csproj

```diff
@@ -23,13 +23,13 @@
     <PackageLicenseFile>LICENSE</PackageLicenseFile>
     <LangVersion>7</LangVersion>
   </PropertyGroup>
-  <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
-  </ItemGroup>
   <ItemGroup>
     <None Include="..\LICENSE">
       <Pack>True</Pack>
       <PackagePath></PackagePath>
     </None>
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
```

Comment:
What about using floating version ? 

https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#floating-versions

Reply:
Floating versions should only be used for patch versions, maybe minor versions, but not major ones. From the documentation:
```The Nearest Wins rule can result in a downgrade of the package version, thus potentially breaking other dependencies in the graph. Hence this rule is applied with a warning to alert the user.```

Newtonsoft adds both bugfixes and new features in patch versions so I do not think it is a good idea to wildcard patch versions for this dependency.
https://github.com/JamesNK/Newtonsoft.Json/releases